bin/npm-publish.js

```diff
@@ -4,9 +4,37 @@ var program = require('commander');
 var fluidPublish = require('fluid-publish');
```

Comment:
Add `use strict` directive.
Using `let` and `const` would be fail on NodeJS `4.x.x` without it.